src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UIPrefsAccessor.java

```diff
@@ -395,7 +395,7 @@ public String getThemeErrorClass()
    
    public PrefValue<Boolean> useDataImport()
    {
-      return bool("use_dataimport", false);
+      return bool("use_dataimport", true);
```

Comment:
You might also need to rename the preference so that anyone who had the false value serialized into their prefs will now have it TRUE.